python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -39,14 +39,13 @@
     from ..schema.util import ResolveInfo
 
 
-def get_run_by_id(
+async def gen_run_by_id(
```

Comment:
name something better than `gen_` prefixed functions

Reply:
Do we not like `gen` any more?

Reply:
it feels like home to me